### PR TITLE
Remove Polar Roles

### PR DESCRIPTION
**Pull request by gj**

### CODE REVIEW

File: languages/python/oso/polar/errors.py [hunk 1]
```diff
@@ -92,10 +89,6 @@ def _operational_error(subkind, message, details):
         return OperationalError(message, details)
 
 
-def _roles_validation_error(message, details):
```
Comment (dhatch) [inline languages/python/oso/polar/errors.py:95]: Do we need to remove these errors from java?

File: languages/ruby/spec/oso/polar/polar_spec.rb [hunk 2]
```diff
@@ -901,7 +901,6 @@ def self.test
       subject.register_class(RolesHelpers::Issue, name: 'Issue')
       subject.register_class(RolesHelpers::User, name: 'User')
       subject.load_file(roles_file)
-      subject.enable_roles
```
Comment (dhatch) [inline languages/ruby/spec/oso/polar/polar_spec.rb:904]: How is the rest of this test still valid?

File: polar-core/src/polar.rs [hunk 3]
```diff
@@ -5,7 +5,6 @@ use super::kb::*;
 use super::messages::*;
```
Comment (dhatch) [inline polar-core/src/polar.rs:5]: Do you need to remove roles errors from the core also? `RolesValidationError`?

File: languages/ruby/spec/oso/polar/polar_spec.rb (hunk 2 above)
Comment (gj) [reply languages/ruby/spec/oso/polar/polar_spec.rb:904]: I updated the 1 giant Polar Roles test in each language to the new syntax

File: languages/python/oso/polar/errors.py (hunk 1 above)
Comment (gj) [reply languages/python/oso/polar/errors.py:95]: I don't see them in Java

File: polar-core/src/polar.rs (hunk 3 above)
Comment (gj) [reply polar-core/src/polar.rs:5]: Done in c507c66

**Review by dhatch — APPROVED** [review]
I did it

**Pull request merged**